### DISCUSSION TITLE
On the project direction

### DISCUSSION THREAD

**Discussion by paspo** [rustdesk/rustdesk#810] [General]
This comment made me think: 
https://github.com/rustdesk/rustdesk/issues/509#issuecomment-1125744953

I completely understand @rustdesk position on the new take, but I don't agree 100% this is the correct way, let me explain.

First of all, I am one of the potential (paying) user base of rustdesk. I do remote assistance everyday and TeamViewer is my solution now. I'm not an happy customer, for many reasons:

* it's not open-source
* I have no control over their servers
* more advanced features requires a license tier that doesn't make sense for a small business

Notice I haven't included the price in this list, because this is a tool for my job and I think 20-30€ /month/seat is a reasonable price for this tool and its features.

I can't switch to Rustdesk right now because I need:

* API access
* (automatic) session recording
* multi-seat addressbook
* custom client executable with embedded server url (the file name solution is a no-go for my case)

I'm writing this because Rustdesk is gaining momentum at the time (beiing the only viable open-source alternative to TeamViewer/Anydesk), a lot of "lower tier" users of other solutions are open to change and looking for an alternative, and we (more or less) think in the same way.

As I said, I understand your idea to reserve some features for paying customers only, but please, consider the simplicity of a dual-licensing: if I use Rustdesk to make money, I need a paying license (with support and all the bells and whistles), otherwise I'm OK with the "free model". No feature or code difference. Only one community, no need to differentiate paying and free users.

My fear is that if you start removing features now (in a moment where I can't buy this product because it misses some features I need), it'll be hard in the future to focus on reliability of such features. If a feature is "behind a paywall" this means less userbase, less testing, less community focus...

I saw some issues in the past because @rustdesk didn't have a raspberrypi on hand to do some test; this happens and the role of the community is helping a developer with these issues. Less users involed means a lower possibility to get help. This time is a raspberrypi, next time maybe you need a hand with a strange kubernetes setup, or worse.

If you start to offer "normal" and "premium" features, it means you have "normal" and "premium" users, "normal" and "premium" products. Are you sure you want to make this distinction?

If I think at other open-source projects I use everyday, like Zabbix and Grafana (we can name a lot of them), their "community edition" is not a "castrated enterprise edition", but a full product.

Hope I didn't offend anyone, I just wanted to share my 2 cents.

**Comment by rustdesk** [comment]
Thanks

**Reply by mwa85** [reply]
@rustdesk

Instead you choice to raise the price last year (with more then 3-times per year increase - $19.9/mth became $69.9/mth) and me and my company dropped RustDesk and went back to TeamViewer as that is the more stable and full-featured product for the same price. I teams like the original vision (dissatisfied with the pricing models, limitations of TeamViewer and AnyDesk) behind RustDesk have been totally lost.

The vision seams to have been totally murder in name of great - and only build on a lie and deceit - and truth came in full after have gain userbase and community on that lie... It's said but now RustDesk true colors are out the last year...

**Comment by kziemski** [comment]
@mwa85 where are you seeing that price increase i thought it was 19/mnth still on the site? 
<img width="191" alt="image" src="https://github.com/user-attachments/assets/e7a77548-e046-40c8-8f97-ac515a61a68a" />
